_docs-sources/foundations/landing-zone/enable-control-tower.md

```diff
@@ -14,6 +16,8 @@ This Guide should take about 1hr 15min to complete, most of that time will be sp
 waiting on Control Tower Operations at the conclusion of the setup flow.
 :::
 
+---
```

Comment:
I was just trying to break the page up a bit. I found that while I was going through these guides and flipping back and forth between the console, it was easy to lose my place. Having some visual breaks always helps me so I chose to put a few of these in. In terms of the overall guide I would agree that they are completely unnecessary, but the visual break made it a little easier for me so I gave it a try. Admittedly there's probably a better way to accomplish this, and I'm okay with removing them if you think it's better without!

Reply:
I can understand the issue of losing one's place. That said, as placed I don't think these align with the content hierarchy particularly well, since they separate the header of one section from the body it pertains to. If anything, it seems like you've "set aside" the pre-requisites section, in a way. I wonder if there's some way we could add a distinct visual treatment to prerequisites _everywhere_ so that they stand out and can be easily identified. That seems out of scope for this change, though…